### PR TITLE
Remove unitstripswarnings

### PR DESCRIPTION
**Pull request by andrewgsavage**
Lazy way to get tests going again

### CODE REVIEW

**Comment by andrewgsavage** [comment]
This should remove most Unitstripwarnings. There's none in the documentation at least! Any thoughts @znicholls ?

Unitstripwarnings notes

They're thrown whenever a method named __array_* is called on a quantity. I've been either avoiding doing this (eg by going to quantity.magnitude) or using with warnings filtered when the warnings won't be an issue.

That being said there's still a few cases where the warning is raised without going through the PintArray, so might need to look at changing how/when the warning is raised.

**Comment by znicholls** [comment]
@andrewgsavage to get tests passing again see https://github.com/andrewgsavage/pint/pull/4

**Comment by znicholls** [comment]
Otherwise I'd rather not silence UnitStripWarnings in the tests, let's leave them if they're there

File: pint/testsuite/pandas_test_interface.py [hunk 1]
```diff
@@ -146,7 +147,35 @@ class TestInterface(base.BaseInterfaceTests):
     pass
 
 class TestMethods(base.BaseMethodsTests):
-    pass
+
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```
Comment (znicholls) [inline pint/testsuite/pandas_test_interface.py:151]: I'd like to keep these warnings popping up unless there's a good reason to silence them?

File: pint/testsuite/pandas_test_interface.py [hunk 2]
```diff
@@ -146,7 +147,35 @@ class TestInterface(base.BaseInterfaceTests):
...
+        else:
+            other = all_data
+
+        result = pd.Series(all_data).value_counts(dropna=dropna).sort_index()
+        expected = pd.Series(other).value_counts(
+            dropna=dropna).sort_index()
+
+        self.assert_series_equal(result, expected)
+    
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```
Comment (znicholls) [inline pint/testsuite/pandas_test_interface.py:167]: I'd like to keep these warnings popping up unless there's a good reason to silence them?

File: pint/testsuite/pandas_test_interface.py [hunk 3]
```diff
@@ -275,8 +304,25 @@ class TestReshaping(base.BaseReshapingTests):
     pass
 
 class TestSetitem(base.BaseSetitemTests):
-    pass
+    @pytest.mark.parametrize('setter', ['loc', None])
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```
Comment (znicholls) [inline pint/testsuite/pandas_test_interface.py:308]: I'd like to keep these warnings popping up unless there's a good reason to silence them?

File: pint/testsuite/pandas_test_interface.py [hunk 4]
```diff
@@ -334,10 +380,15 @@ def test_pintarray_creation(self):
         for y in ys:
             self.assertQuantityAlmostEqual(x,y.data)
 
+    
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```
Comment (znicholls) [inline pint/testsuite/pandas_test_interface.py:384]: I'd like to keep these warnings popping up unless there's a good reason to silence them?

File: pint/testsuite/pandas_test_interface.py (hunk 1 above)
Comment (andrewgsavage) [reply pint/testsuite/pandas_test_interface.py:151]: The ones I've been silencing should be occurring without going through the PintArray, so there's nothing we can do in pint_array.py to get rid of them - That'd have to be done in quantity.py.

Main reason I'd been silencing them was to get through them all easier. I think it's easier to spot whether a future change starts creating a UnitStrippedWarning like this.

I'm not fussed either way.

Comment (znicholls) [reply pint/testsuite/pandas_test_interface.py:151]: ah ok makes sense, if they're not an issue then we leave it for now

**Comment by andrewgsavage** [comment]
So is this OK to merge then?

**Comment by znicholls** [comment]
yep sorry

**Comment by andrewgsavage** [comment]
Does it's time to get this into pint then? Exciting!

**Comment by znicholls** [comment]
> Does it's time to get this into pint then?

ya

**Pull request merged**